### DISCUSSION TITLE
Does it works on computers/company restristing USB usage for Keyboard/Mouse only ?

### DISCUSSION THREAD

**Discussion by OrpheeGT** [hrvach/deskhop#38] [General]
Hi !
Title say it all.
On a Windows using GPO to lock down USB for only Keyboard/mouse ? will it still work ?
Some company disable USB ports for USB storage etc...
Thanks!

**Comment by hrvach** [comment]
It should work, as long as ports not disabled for HID devices (keyboard/mouse).

**Comment by OrpheeGT** [comment]
So I can answer myself, it works ! :)
